conf/deployment/aws/gp3_csi_storage_class.yaml

```diff
@@ -0,0 +1,4 @@
+---
+# This config is for AWS deployment with gp3 CSI storage class provisioning
+DEPLOYMENT:
+  customized_deployment_storage_class: 'gp3-csi'
```

Comment:
the `conf/deployment/{platform}` is the directory where we store whole production jobs configurations which are suppose to contains all details about deployment + usually polarion ID.

This is just a chagne in deployment which probably should be just under `conf/ocsci` folder.
The same for the above. @yitzhak12 can you please move those two files?